app/core/main.js

```diff
@@ -44,6 +45,10 @@ mb.on('ready', () => {
   ipcMain.on('delete-log', (event, arg) => {
     fs.writeFile(logPath, arg);
   });
+  ipcMain.on('set-service-hook', (event, arg) => {
+    setting.enableServiceHook = arg;
+    fs.writeFile(settingPath, JSON.stringify(setting));
```

Comment:
You can use pretty print
```
JSON.stringify({ uno: 1, dos: 2 }, null, '  ');
// returns the string:
// '{
//   "uno": 1,
//   "dos": 2
// }'
```
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify